### PR TITLE
Add new text, images, divs

### PR DESCRIPTION
**Pull request by tjcrone**

### CODE REVIEW

**Comment by tjcrone** [comment]
it's possible to put divs with css style directly in Markdown, which is how i got the figures to look the way they do. this is not ideal if the site gets large. we should probably figure out how to add elements to the style sheet so that changes to the style will be easy in the future.

File: sensor_systems.md [hunk 1]
```diff
@@ -4,32 +4,52 @@ title: Sensor System Development
...
+### Now
 
+The Real-Time Earth initiative is grounded in the idea that observations drive scientific discovery, and that the Earth sciences are on the threshold of a real-time scientific revolution supported by the proliferation of networked sensors, instruments, devices, and platforms. These sensor systems will enable innovative scientific investigations spanning a large range of spatial and temporal scales and give scientists unprecedented insights into numerous dynamic and interconnected processes on this planet. Lamont scientists have been leading the nascent real-time science revolution. Here are a just a few examples of their groundbreaking research in this field:
 
-#### Examples of Sensor Engineering at Lamont
+#### Cabled Observatories
 
-**Cabled Observatories**
+<div style="color:#888888; font-size: 7pt; width:300px; line-height:1.2em; float:left;
+margin-right:20px; margin-top:8px">
```
Comment (rabernat) [inline sensor_systems.md]: Have you considered putting all this CSS into the CSS file (css/main.scss) as a new class? Then you could just say

```html
<div class="special">
Foo bar
</div>

Comment (tjcrone) [inline sensor_systems.md]: that would be good. i wasn't sure what the jekyll jazz was going to clobber, or what is really happening when it does it's magic. but if we can add to main.scss, then i will try that.

Comment (rabernat) [inline sensor_systems.md]: Yeah it uses [SASS](http://sass-lang.com/) to template the stylesheet I think. Not sure at what point the template gets compiled. But I think you should be able to edit the main.scss file.

**Pull request merged**